pandas/tests/test_internals_external_block.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=W0102
+
+import numpy as np
+
+import pandas as pd
```

Comment:
maybe make a sub-dir for internals tests ``tests/internals/test_internals.py``, ``test_block`` or some-such